app/src/main/java/com/github/se/gomeet/MainActivity.kt

```diff
@@ -123,7 +123,7 @@ class MainActivity : ComponentActivity() {
                       if (result.isSuccess) {
                         NavigationActions(nav)
                             .navigateTo(
-                                TOP_LEVEL_DESTINATIONS.first { it.route == Route.CREATE },
+                                TOP_LEVEL_DESTINATIONS.first { it.route == Route.EXPLORE },
```

Comment:
this breaks end to end tests since they assume the app starts in create. i've been trying to fix them but i'm getting weird errors